Client/src/app/components/authentication/login/login.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="background" [ngStyle]="{'background-color': bgColor}">
+  <div class="otherServices">
+    <button class="button">
+      <p class="content" [style.color]="fgColor">{{goggleBtnText}}</p>
+    </button>
+    <button class="button">
+      <p class="content" [style.color]="fgColor">{{facebookBtnText}}</p>
+    </button>
+  </div>
+  <hr class="verticalLine" [style.border-color]="fgColor" />
+
+  <div class="loginArea" [style.color]="fgColor">
+    <form>
+      <label for="username">Username/Email</label>
+      <input type="text" [(ngModel)]="usernameEmail" name="username" id="username" class="inputField" [style.color]="fgColor" required />
+      <label for="password">Password</label>
+      <input type="text" [(ngModel)]="password" name="password" id="password" class="inputField" [style.color]="fgColor" required />
```

Comment:
add some more validation so the request should be successfull if it hits our server. I think this should also be a type of password.

Reply:
And what kind of validations?

Reply:
And what kind of validations